### PR TITLE
Documentation and install updates

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ articles:
     - create-individual
     - create-run-population
     - observed-data
+    - data-combined
+    - data-combined-plotting
```
Comment (Yuri05) [inline _pkgdown.yml:28]: should we add a separate group for these 2 vignettes (e.g. "Plotting")?

Comment (PavelBal) [reply _pkgdown.yml:32]: Yes good idea.

File: README.md [hunk 2]
```diff
@@ -146,7 +152,26 @@ vignette("introduction-ospsuite")
...
+Error: package or namespace load failed for ‘ospsuite’:
+loadNamespace()里算'ospsuite'时.onLoad失败了，详细内容：
+调用: rClr::clrCall(dimensionTask, "AllAvailableUnitNamesFor", enc2utf8(dimension))
+错误: Type: System.Collections.Generic.KeyNotFoundException
+Message: Dimension 'CV mmHg*s虏/ml' not available in DimensionFactory.
+...
+```
+
+On Windows, set `Settings > Language > Administrative language settings > Current language for non-Unicode programs` 
+to `English (United States)` and reboot.
```
Comment (Yuri05) [inline README.md:172]: on Linux: set the environment variable **LC_ALL** before starting R:
`export LC_ALL=en_US.UTF-8`

File: inst/extdata/install-packages.R [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
-  RToolsNotFound = function(rToolsPath) {
-    paste0("Rtools not found at ", rToolsPath, ", cannot continue")
-  },
+  installRTools = "Install Rtools compatible with your R version, then run updateEnvironment() again",
   PKSimLoadFails = "PK-Sim fails to load. The installation might be incompatible with your current version of PK-Sim"
 )
 
 # Packages that will be installed from CRAN
-.cranPackages <- c("R6", "stringr", "readr", "readxl", "shiny", "dplyr", "tidyr", "ggplot2", "purr", "patchwork", "jsonlite")
+.cranPackages <- c("R6", "stringr", "readr", "readxl", "shiny", "dplyr", "tidyr", "ggplot2", "purrr", "patchwork", "jsonlite")
```
Comment (Yuri05) [inline inst/extdata/install-packages.R:8]: lol

File: inst/extdata/install-packages.R [hunk 4]
```diff
@@ -75,47 +72,45 @@ displayProgress <- function(current, success = TRUE, message = NULL, suppressOut
...
+  install.packages(.cranPackages,
+    dependencies = TRUE,
+    lib = lib
+  )
 
   displayProgress("Installing rClr", suppressOutput = suppressOutput)
   if (version$major == "4") {
-    install.packages(paste0("https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v", rclrVersion, "/rClr_", rclrVersion, ".zip"), repos = NULL)
+    install.packages(paste0("https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v", rclrVersion, "/rClr_", rclrVersion, ".zip"), repos = NULL,
+                     lib = lib)
```
Comment (Yuri05) [inline inst/extdata/install-packages.R:109]: `...` is not passed as argument

File: inst/extdata/install-packages.R [hunk 5]
```diff
@@ -75,47 +72,45 @@ displayProgress <- function(current, success = TRUE, message = NULL, suppressOut
...
   }
   if (version$major == "3") {
-    install.packages(paste0("https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v", rclrVersion, "-R3/rClr_", rclrVersion, ".zip"), repos = NULL)
+    install.packages(paste0("https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v", rclrVersion, "-R3/rClr_", rclrVersion, ".zip"), repos = NULL,
+                     lib = lib)
```
Comment (Yuri05) [inline inst/extdata/install-packages.R:113]: `...` is not passed as argument

File: inst/extdata/install-packages.R [hunk 6]
```diff
@@ -124,11 +119,11 @@ installOSPPackages <- function(rtoolsPath = NULL, rclrVersion = "0.9.2",
   }
 
   displayProgress("Installing ospsuite.utils", suppressOutput = suppressOutput)
-  install.packages(packagePaths$ospsuite.utils, repos = NULL)
+  install.packages(packagePaths$ospsuite.utils, repos = NULL, lib = lib, ...)
   displayProgress("Installing tlf", suppressOutput = suppressOutput)
   install.packages(packagePaths$tlf, repos = NULL)
```
Comment (Yuri05) [inline inst/extdata/install-packages.R:124]: `...` is not passed as argument

File: inst/extdata/install-packages.R (hunk 4 above)
Comment (PavelBal) [reply inst/extdata/install-packages.R:109]: Did you ever run the tests for `rClr`? I would not risk it as we have no idea what there is and if it actually works :D


This is when I try to run them:

==> devtools::test()

```
ℹ Testing rClr
Error in nsenv[[f_name]](dirname(ns_path), package) : 
  rClr library .NET framework not found - looked under D:/Work/OSPS/Repos/rClr/inst/libs//x64 but not found in 
Calls: <Anonymous> ... test_files_setup_env -> <Anonymous> -> run_pkg_hook -> <Anonymous>
Execution halted

Exited with status 1.
```

File: README.md (hunk 2 above)
Comment (PavelBal) [reply README.md:172]: Added

File: inst/extdata/install-packages.R (hunk 6 above)
Comment (PavelBal) [reply inst/extdata/install-packages.R]: Fixed

File: README.md (hunk 2 above)
Comment (PavelBal) [reply README.md:172]: Added

File: _pkgdown.yml (hunk 1 above)
Comment (PavelBal) [reply _pkgdown.yml:32]: A new group now.

File: inst/extdata/install-packages.R (hunk 4 above)
Comment (Yuri05) [reply inst/extdata/install-packages.R:109]: Well, installing the tests does not mean they are automatically run (and windows .zip packages do not contain any tests anyways - unfortunately).
But I mean, we can skip this part for rClr (I always run the tests for tlf, ospsuite.utils, ospsuite, ospsuite.reportingengine - but indeed not for rClr :)

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**